test/Microsoft.Azure.Devices.Edge.Test.TempSensor/Program.cs

```diff
@@ -110,4 +116,26 @@ protected override ValidationResult IsValid(object value, ValidationContext cont
             return ValidationResult.Success;
         }
     }
+
+    [AttributeUsage(AttributeTargets.Class)]
```

Comment:
Nit: Suggest to move these validation attribute classes into its own files; since these attributes are supposed to be used in other tests as well.

Reply:
I'm already doing that work in my next PR because it actually adds a 2nd test.